github3/pulls.py

```diff
@@ -208,6 +209,16 @@ def close(self):
         return self.update(self.title, self.body, 'closed')
 
     @requires_auth
+    def create_comment(self, body):
+        """Create a comment on this pull request's issue.
+
+        :param str body: (required), comment body
+        :returns: :class:`IssueComment <github3.issues.comment.IssueComment>`
+        """
+        issue = self.issue()
```

Comment:
So, I've actually been trying to move away from having methods that make more than one request.

What we can do, however, is something like

```py
url = self.links['comments']['href']
json = None
if body:
    json = self._json(self._post(url, data={'body': body}),
                      201)
return self._instance_or_null(IssueComment, json)
```